app/mailers/attendance_mailer.rb

```diff
@@ -29,30 +29,27 @@ def custom_first_reminder(attendance_id)
     cancel_delivery unless @attendance
 
     @user = @attendance.user
-    tt = @attendance.tea_time
-
-    body = tt.host.email_reminder.body
-
-    cancel_delivery unless body
+    @tt = @attendance.tea_time
 
-    body += "\n\n <hr> *Note from the Robots: This email is about your tea time on #{tt.date_to_email}. It's at #{tt.location}. Enjoy it!*"
+    @body = @tt.host.email_reminder.body
 
-    # See here for options https://github.com/vmg/redcarpet
-    renderer = Redcarpet::Render::HTML.new(filter_html: true, hard_wrap: true, escape_html: true)
-    markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
-    sanitized_body = markdown.render(body)
+    cancel_delivery unless @body
 
     attachments['event.ics'] = {
       mime_type: "text/calendar",
       content: IcalCreator.new(@attendance.tea_time).call.to_ical
     }
+
     cancel_delivery unless @attendance.pending?
+
     mail(
       to:           @attendance.user.email,
-      from:         "\"#{tt.host.nickname} at Tea With Strangers\" <#{tt.host.email}>",
+      from:         "\"#{@tt.host.nickname} at Tea With Strangers\" <#{@tt.host.email}>",
       subject:      "Your tea time is coming up!",
-      body:         sanitized_body,
-      content_type: "text/html")
+    ) do |format|
+      format.text
+      format.html
+    end
   end
 
   # Twelve-hour and two day reminder of a tea time a user is attending
```

Comment:
yep! the default one is sent if they haven't customized one. See the [`queue_first_reminder`](https://github.com/TeaWithStrangers/tws-on-rails/blob/master/app/models/attendance.rb#L60-L64) to see how it gets scheduled.